x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -62,23 +62,45 @@ curl --cacert config/certs/http_ca.crt -u elastic https://localhost:9200
 ----
 // NOTCONSOLE
 
-. From the directory where you installed {kib}, start {kib}.
+. From the directory where you installed {kib}, start {kib} in either
+interactive or detached mode.
 +
+--
+*Interactive mode*
+
+Use this mode if you want to enroll {kib} using your browser.
+
 [source,shell]
 ----
 bin/kibana
 ----
-+
+
 This command generates a unique link to enroll your {kib} instance with {es}.
 
-  .. In your terminal, click the generated link to open {kib} in your browser.
+. In your terminal, click the generated link to open {kib} in your browser.
 
-  .. In your browser, paste the enrollment token that you copied and click the
+. In your browser, paste the enrollment token that you copied and click the
 button to connect your {kib} instance with {es}.
 
-  .. Log in to {kib} as the `elastic` user with the password that was generated
+. Log in to {kib} as the `elastic` user with the password that was generated
 when you started {es}.
 
+'''
+
+*Detached mode*
+
+Use this mode if you want to enroll {kib} without a browser, or want to enroll
+{kib} on a machine where you don't have access to a browser.
+
+Start {kib} and pass the generated enrollment token with the `--token`
+parameter.
+
+[source,shell]
+----
+bin/kibana_setup --token <enrollment-token>
```

Comment:
The command and the argument here are wrong 😅 

```suggestion
bin/kibana-setup --enrollment-token <enrollment-token>
```

Reply:
Thanks @jportner! I incorporated this change in my latest push.